src/styles/watch.scss

```diff
@@ -0,0 +1,63 @@
+.watch {
+  height: 500px;
+  width: 500px;
+  border: 7px dotted #000;
+  border-radius: 50%;
+  box-sizing: border-box;
+  position: relative;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%);
+
+  &__center {
+    position: absolute;
+    background-color: yellow;
+    width: 10%;
+    height: 10%;
+    border: 1px solid yellow;
+    border-radius: 50%;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+  }
+  &__minute-hand {
+    position: absolute;
+    background-color: green;
+    height: 50%;
+    width: 10px;
+    border-radius: 30%;
+
+    left: 50%;
+    transform: translate(-50%);
+    z-index: -1;
+    animation-duration: 5s;
+    animation-iteration-count: infinite;
+    @include rotate();
+    transform-origin: bottom;
```

Comment:
The 'transform-origin: bottom;' property should be 'transform-origin: center bottom;' to correctly rotate the minute hand around its bottom center point.